WordPress/Classes/Networking/CommentServiceRemoteXMLRPC.m

```diff
@@ -23,9 +23,20 @@ - (void)getCommentsForBlog:(Blog *)blog
                    success:(void (^)(NSArray *))success
                    failure:(void (^)(NSError *))failure
 {
-    NSDictionary *extraParameters = @{
-                                     @"number": @100
-                                     };
+    [self getCommentsForBlog:blog options:nil success:success failure:failure];
+}
+
+- (void)getCommentsForBlog:(Blog *)blog
+                   options:(NSDictionary *)options
+                   success:(void (^)(NSArray *))success
+                   failure:(void (^)(NSError *))failure
+{
+    NSMutableDictionary *extraParameters = [NSMutableDictionary dictionaryWithDictionary:@{
+                                      @"number": @100
```

Comment:
I realize it was this way before, but maybe make the number a const along with the rest of the changes?